lib/tasks/build.rake

```diff
@@ -3,12 +3,12 @@ TAILWIND_COMPILE_COMMAND = "#{RbConfig.ruby} #{Pathname.new(__dir__).to_s}/../..
 namespace :tailwindcss do
   desc "Build your Tailwind CSS"
   task :build do
-    exec TAILWIND_COMPILE_COMMAND
+    system(TAILWIND_COMPILE_COMMAND, exception: true)
   end
 
   desc "Watch and build your Tailwind CSS on file changes"
   task :watch do
-    exec "#{TAILWIND_COMPILE_COMMAND} -w"
+    system "#{TAILWIND_COMPILE_COMMAND} -w"
```

Comment:
Is there a reason `exception: true` is not used here? Even though the command is not expected to terminate, it can still fail to start in the first place.

Reply:
No reason other than it hasn't been a problem we needed to solve. If you'd like to submit a PR I'd consider merging it.